backend/src/migrations/20240820093010_add_name_field_to_users_table.ts

```diff
@@ -0,0 +1,15 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'users';
+
+async function up(knex: Knex): Promise<void> {
+    await knex.schema.table(TABLE_NAME, (table) => {
+        table.string('name').notNullable();
+    });
+}
+
+function down(knex: Knex): Promise<void> {
+    return knex.schema.dropTableIfExists(TABLE_NAME);
```

Comment:
Just drop the field not the table

Reply:
tanks for the observation, I'll change it.

Reply:
thanks Anton, I fixed it.